### PR TITLE
Initial Setup of Node Project

### PR DESCRIPTION
**Pull request by Maosorio92**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @holgiosalos @danielgalvis98 @kliver98 @AlejaGonzalezV @NicolasB2 @manuelq12 @Valeryibarra @veronicatofino
```
Comment (danielgalvis98) [inline .github/CODEOWNERS]: Add End of File (EOF)

File: test/HelloWorld.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const { assert } = require('chai');
+
+describe('Array', () => {
+  describe('#indexOf()', () => {
+    it('should return -1 when the value is not present', function() {
+      assert.equal(-1, [1,2,3].indexOf(4));
+    });
+  });
+});
```
Comment (danielgalvis98) [inline test/HelloWorld.test.js]: Add EOF

**Review by danielgalvis98 — CHANGES_REQUESTED** [review]
Also add the description to the Readme

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+*.cab
+*.msi
+*.msix
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
+
+# End of https://www.toptal.com/developers/gitignore/api/windows,visualstudiocode,node
```
Comment (NicolasB2) [inline .gitignore]: Add EOF

File: .github/CODEOWNERS (hunk 1 above)
Comment (Maosorio92) [reply .github/CODEOWNERS]: Thank you!

File: test/HelloWorld.test.js (hunk 2 above)
Comment (Maosorio92) [reply test/HelloWorld.test.js]: Ok

File: .gitignore (hunk 3 above)
Comment (Maosorio92) [reply .gitignore]: Ok, thank you

File: README.md [hunk 4]
```diff
@@ -1 +1,2 @@
+This is the first workshop about API testing by Manuel Osorio
```
Comment (danielgalvis98) [inline README.md]: Put this as the second line and with "##"
```suggestion
# workshop-api-testing-js
## This is the first workshop about API testing by Manuel Osorio
```

**Review by danielgalvis98 — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,2 +1,3 @@
-This is the first workshop about API testing by Manuel Osorio
+# workshop-api-testing-js
+## This is the first workshop about API testing by Manuel Osorio
 # workshop-api-testing-js
```
Comment (danielgalvis98) [inline README.md]: Remove this line

Comment (Maosorio92) [reply README.md]: Hello!
I have a question, yesterday, I accepted the first change you made in readme.md file, then, you suggested me to erase a line, but now, I don´t know how to accept this change, in fact, now I don´t see the line you suggested me to erase. I am confused.

Comment (Maosorio92) [reply README.md]: Or is it better for me tu push the file again?

Comment (veronicatofino) [reply README.md]: @Maosorio92 The third line is the one to remove. You are repeating it from the first line.

Comment (danielgalvis98) [reply README.md]: @Maosorio92 As Veronica says, the line I'm referring to is the third line, that has `# workshop-api-testing-js` repeated.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+    "type-detect": "^4.0.8",
+    "workerpool": "^6.2.1",
+    "wrap-ansi": "^7.0.0",
+    "wrappy": "^1.0.2",
+    "yargs": "^16.2.0",
+    "y18n": "^5.0.8",
+    "yargs-parser": "^20.2.4",
+    "yargs-unparser": "^2.0.0",
+    "yocto-queue": "^0.1.0"
+  }
```
Comment (holgiosalos) [inline package.json]: are these dependencies required? I think that just mocha and chai as dev dependencies are required for this step

**Review by manuelq12 — APPROVED** [review]

**Review by danielgalvis98 — APPROVED** [review]

**Pull request merged**